### PR TITLE
Icon classnames

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by danni** [comment]
Add the sorting while you're at it.

**Comment by DanielHeath** [comment]
done

**Comment by danni** [comment]
++ assuming you unbadgered the tests

**Comment by DanielHeath** [comment]
Tests are suffering from concurrency being greater than our saucelabs account allows (5 builds at a time)

**Comment by DanielHeath** [comment]
Tests were also failing because icons were clobbering any className you passed in. Fixed now.

File: src/icons/ChevronBack.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* @flow */
+/* jscs: disable */
+/* This code is generated by ./script/iconify - do not edit. */
+
+import React from "react";
+import mui from "material-ui";
+
+export default class SvgIconChevronBack extends React.Component {
+
+    render(): React.Component {
```
Comment (danni) [inline src/icons/ChevronBack.js:10]: These are actually `React.Element`

**Comment by danni** [comment]
Also you should check for any existing uses of icon names as className in the code and remove it.

**Pull request merged**